server/src/app.js

```diff
@@ -1,17 +1,20 @@
 const express = require('express');
+const cookieParser = require('cookie-parser');
 const cors = require('cors');
 
-const path = require('path');
 const bodyParser = require('body-parser');
 const routes = require('./controllers');
 require('dotenv').config();
 
 const app = express();
 
 app.set('port', process.env.PORT || 4000);
-app.use(cors());
+app.use(cookieParser());
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
+app.use(cors({origin:true,credentials: true}));
```

Comment:
I see glad it worked, however with this cors configuration , anyone can use your server to get/post data ?

Reply:
@LogicX1 close your eyes when you delve into this PR, it's chaos

Reply:
> I see glad it worked, however with this cors configuration , anyone can use your server to get/post data ?

technically yes, think about it as an open API, you just have to secure certain routes that you want secured, that's why we have the `authCheck` middleware

Reply:
I see well if it works it works!
We were saved of this hassle by using auth0, though I still wasted a lot of time reading their documentation and tutorials